pkg/processor/sourceprocessor/README.md

```diff
@@ -4,42 +4,74 @@ The `sourceprocessor` adds `_sourceName` and other tags related to Sumo Logic me
 
 It is recommended to use `k8sprocessor` to provide attributes used in default values.
 
-## Config
+## Configuration
 
-- `collector` (default = ``): name of the collector, put in `_collector` tag
-- `source_name` (default = `%{k8s.namespace.name}.%{k8s.pod.name}.%{k8s.container.name}`): `_sourceName` template
-- `source_category` (default = `%{k8s.namespace.name}/%{k8s.pod.pod_name}`): `_sourceCategory` template
-- `source_category_prefix` (default = `kubernetes/`): prefix added before each `_sourceCategory` value
-- `source_category_replace_dash` (default = `/`): character which all dashes (`-`) are being replaced to
-
-### Filtering section
-
-**NOTE**: The filtering is done on the resource level attributes.
-
-- `exclude` (default = `{}`): a mapping of field names to exclusion regexes
-  for those particular fields. Whenever a value under particular field matches
-  a corresponding regex, the processed entry is dropped.
-
-### Keys section
-
-The following keys must match resource attributes.
-In most cases the keys should be the same like in [k8sprocessor](../k8sprocessor/README.md#extract-section) config:
+```yaml
+processors:
+  source:
+    # Name of the collector, put in `_collector` tag.
+    # default: ""
+    collector: <collector>
+    # Template for source name, put in `_sourceName` tag.
+    # default: "%{k8s.namespace.name}.%{k8s.pod.name}.%{k8s.container.name}"
+    source_name: <source_name>
+    # Template for source category, put in `_sourceCategory` tag.
+    # default: "%{k8s.namespace.name}/%{k8s.pod.pod_name}"
+    source_category: <source_category>
+    # Prefix added before each `_sourceCategory` value.
```

Comment:
`except container annotations`?

Reply:
If they've read about container annotations, they already know it overwrites this setting.

If they haven't read about container annotations, it doesn't matter, because they are not using them.